testdata/local/admin.json

```
@@ -1,4 +1,4 @@
-{ "_id" : "ui", "default_project" : "evergreen", "url" : "http://localhost:9090", "http_listen_addr" : ":9090", "secret" : "this is a secret", "cors_origins": ["http://localhost:3000","http://localhost:5173","http://localhost:4173"], "userVoice": "https://uservoice.com", "uiv2_url": "http://localhost:3000", "parsley_url": "http://localhost:4173"}
+{ "_id" : "ui", "default_project" : "evergreen", "url" : "http://localhost:9090", "http_listen_addr" : ":9090", "secret" : "this is a secret", "cors_origins": ["http://localhost:3000","http://localhost:5173","http://localhost:4173"], "valid_content_types": ["application/json", "text/plain", "text/csv"] "userVoice": "https://uservoice.com", "uiv2_url": "http://localhost:3000", "parsley_url": "http://localhost:4173"}
```

nit. missing a comma, invalid JSON

```suggestion
{ "_id" : "ui", "default_project" : "evergreen", "url" : "http://localhost:9090", "http_listen_addr" : ":9090", "secret" : "this is a secret", "cors_origins": ["http://localhost:3000","http://localhost:5173","http://localhost:4173"], "valid_content_types": ["application/json", "text/plain", "text/csv"], "userVoice": "https://uservoice.com", "uiv2_url": "http://localhost:3000", "parsley_url": "http://localhost:4173"}
```